core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.html

```diff
@@ -0,0 +1,161 @@
+<div class="oppia-edit-thumbnail-container">
+  <div class="modal-header">
+    <h3>Upload Thumbnail</h3>
+  </div>
+
+  <div class="modal-body oppia-thumbnail-uploader-container">
+    <div class="oppia-thumbnail-uploader">
+      <div [hidden]="uploadedImage">
+        <strong>Please upload an SVG image.</strong>
+        <div class="alert alert-warning">
+          <span>Here are some guidelines to follow:</span>
+          <ul>
+            <li>Uploaded image should have an aspect ratio of {{ aspectRatio }}.</li>
+            <li>Uploaded image should have a transparent background.</li>
+            <li>Please ensure that the uploaded image is less than 100 KB.</li>
+          </ul>
+        </div>
+        <oppia-image-uploader (fileChanged)="onFileChanged($event)"
+                              [allowedImageFormats]="allowedImageFormats">
+        </oppia-image-uploader>
+      </div>
+      <div class="oppia-form-error" *ngIf="invalidImageWarningIsShown">
+        Please upload a valid SVG image.
+      </div>
+      <div class="oppia-form-error" *ngIf="invalidTagsAndAttributes?.tags.length">
+        Unsupported SVG tags found: {{ invalidTagsAndAttributes.tags }}
+      </div>
+      <div class="oppia-form-error" *ngIf="invalidTagsAndAttributes?.attrs.length">
+        Unsupported SVG attributes found: {{ invalidTagsAndAttributes.attrs }}
+      </div>
+      <div [hidden]="!uploadedImage">
+        <div [hidden]="!allowedBgColors">
+          <div [hidden]="!(allowedBgColors.length > 1)">Choose background color:</div>
+          <div [hidden]="!(allowedBgColors.length > 1)" class="oppia-thumbnail-colour-select">
+            <div *ngFor="let color of allowedBgColors"
+                 [ngStyle]="{'background': color}"
+                 (click)="updateBackgroundColor(color)">
+            </div>
+          </div>
+          <div [hidden]="!(allowedBgColors.length === 1)" class="alert alert-warning">
+            <span>
+              Please check that the thumbnail below has the following background color:
+              <i class="fa fa-square" [ngStyle]="{'color': allowedBgColors[0]}"></i>.
+              If not, please ensure that you are uploading an image with transparent background. Thanks!
+            </span>
+          </div>
+        </div>
+        <div @fade class="oppia-thumbnail-container protractor-test-thumbnail-container"
+             *ngIf="uploadedImage"
+             [ngStyle]="{'background': previewDescriptionBgColor, 'color': previewDescriptionBgColor ? '#FFFFFF' : '', 'width': aspectRatio === '4:3' ? '248px' : '320px' }">
+          <button class="btn btn-secondary oppia-thumbnail-reset-button protractor-thumbnail-reset-button" (click)="reset()">
+            <i class="material-icons oppia-vcenter">&#xE14C;</i>
+          </button>
+          <div>
+            <oppia-thumbnail-display
```

Comment:
please move ngIf to the same line as tag name

Reply:
done

Reply:
Is it done? I still see ngIf on line 55 while <oppia-thumbnail-display is on line 54.